WordPress/gutenberg
# UnitControl: Fix an issue where keyboard shortcuts unintentionally shift focus on Windows OS

Follow-up #39303
Fixes #62967

## What?

This PR fixes an issue where the focus unintentionally moves to the unit when executing the paste shortcut (`Ctrl+V`) in an input field on Windows OS.

## Why?
Only `event.metaKey` is checked to give focus to the unit, which may work on MacOS, but on Windows OS the `Ctrl` key is `event.ctrlKey`.


## Testing Instructions

The following are test procedures for Windows OS, but they should continue to work as expected on MacOS.

- Run the Storybook.
- Access `localhost:50240/?path=/docs/components-experimental-unitcontrol--docs`
- Copy the numbers beforehand.
- Run `Ctrl+V` in the input field.
- The copied numbers should be pasted.
- Press only the `p`, `v` or `e` key.
- The focus should move to the unit, and the unit should change according to the key pressed.

## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/54422211/536a7482-90e5-4ebd-98ab-83557b403626
